CHANGELOG.md

```diff
@@ -1,5 +1,27 @@
 # Changelog
 
+## [2.3.2](https://launchpad.net/mixxx/+milestone/2.3.3) (unreleased)
+
+* Pioneer-DDJ-SB3: fix shift button release [#4659](https://github.com/mixxxdj/mixxx/pull/4659)
+* Traktor S3: Push two deck switches to explicitly clone decks [#4665](https://github.com/mixxxdj/mixxx/pull/4665) [#4671](https://github.com/mixxxdj/mixxx/pull/4671) [lp:1960680](https://bugs.launchpad.net/mixxx/+bug/1960680)
+* Behringer DDM4000: Update controller mapping [#4318](https://github.com/mixxxdj/mixxx/pull/4318)
+* MC7000: Fix 'inverted shift' bug in the controller mapping [#4755](https://github.com/mixxxdj/mixxx/pull/4755)
+* Fix spinback and break effect in the controller engine [#4708](https://github.com/mixxxdj/mixxx/pull/4708)
+* Fix scratch on first wheel touch [#4761](https://github.com/mixxxdj/mixxx/pull/4761) [lp:1800343](https://bugs.launchpad.net/mixxx/+bug/1800343)
+* Pref controller: prevent false-positive 'dirty' flag [#4721](https://github.com/mixxxdj/mixxx/pull/4721) [lp:1920844](https://bugs.launchpad.net/mixxx/+bug/1920844)
+* Fix rare SEGFAULT when closing the progress dialog [#4695](https://github.com/mixxxdj/mixxx/pull/4695)
+* Prevent preferences dialog from going out of screen [#4613](https://github.com/mixxxdj/mixxx/pull/4613)
+* Fix undesired jump-cuts in Auto-DJ [#4693](https://github.com/mixxxdj/mixxx/pull/4693) [lp:1948975](https://bugs.launchpad.net/mixxx/+bug/1948975) [lp:1893197](https://bugs.launchpad.net/mixxx/+bug/1893197)
+* Fix stopped Auto-DJ [#4698](https://github.com/mixxxdj/mixxx/pull/4698) [lp:1893197](https://bugs.launchpad.net/mixxx/+bug/1893197) [lp:1961970](https://bugs.launchpad.net/mixxx/+bug/1961970)
```

Comment:
This sounds wrong. How about "Fix bug that caused Auto DJ to stop playback after some time"